src/js/tech/tech.js

```diff
@@ -141,6 +141,8 @@ class Tech extends Component {
       this.emulateTextTracks();
     }
 
+    this.preloadTextTracks = (options.preloadTextTracks !== false);
```

Comment:
This may not be necessary, we can just get the option directly in the text-track file.

Reply:
applying...